platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/StateConfig.java

```diff
@@ -113,7 +113,8 @@ public record StateConfig(
         @ConfigProperty(defaultValue = "5") int debugHashDepth,
         @ConfigProperty(defaultValue = "1000") int maxAgeOfFutureStateSignatures,
         @ConfigProperty(defaultValue = "26") int roundsToKeepForSigning,
-        @ConfigProperty(defaultValue = "false") boolean signedStateSentinelEnabled) {
+        @ConfigProperty(defaultValue = "false") boolean signedStateSentinelEnabled,
+        @ConfigProperty(defaultValue = "false") boolean forceUseOfConfigAddressBook) {
```

Comment:
shouldnt the default be true?

Reply:
After conversation with @cody-littley, I'm bought in on making it true to begin with to leave the capability disabled by default until sufficient testing has happened to switch it to false.   I'll change the value in the next update.